Automattic/jetpack
# Blogroll: create blocks from recommendations

This PR creates the inner blocks ( blogroll items ) based on the response received by the recommendations endpoint.

p9Jlb4-8nF-p2#adding-the-block

Fixes #
https://github.com/Automattic/wp-calypso/issues/81351

## Proposed changes:
* Fetch recommendations and use them to create inner blocks
* Create inner template for blogroll item
* Styling fixed 

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to  the editor
* Add the blogroll block
* Ensure that the response from `recommendations` is used to create the initial content.
